CHANGELOG.md

```diff
@@ -780,7 +784,7 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Danielle Tomlinson](https://github.com/dantoml)
   [#5937](https://github.com/CocoaPods/CocoaPods/issues/5937)
 
-* Move Installer target verification into the Xcode namespace 
+* Move Installer target verification into the Xcode namespace
   [Danielle Tomlinson](https://github.com/DanToml)
   [#5607](https://github.com/CocoaPods/CocoaPods/pull/5607)
 
```

Comment:
Any chance you can undo these whitespace changes and re-add your entry with format on save disabled? (if you're using vscode that's `cmd + alt + shift + s` )

Reply:
re-pushed the clean changes, thank you. will check out the vscode.